kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/extensions/IsActiveExtension.kt

```diff
@@ -2,6 +2,7 @@ package io.kotest.core.extensions
 
 import io.kotest.core.config.ExperimentalKotest
 import io.kotest.core.plan.Descriptor
+import io.kotest.core.test.IsActive
```

Comment:
I'm not sure "active" is the right word.
We use active in a couple of functions and this listener but I think enabled is used mostly.
So what about IsEnabled or just Enabled and perhaps we rename this listener too?